rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,335 @@
+//---
+//date: '2023-01-03T22:07:08.574151'
+//title: AWS Load Balancer Operator On ROSA
+//aliases: ['/docs/rosa/alb-sts']
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Shaozhen Ding
+//  - Paul Czarkowski
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+
+[TIP]
+====
+AWS Load Balancer Operator cannot be used for {product-title} Routes, thus should only be used for individual services or Ingress that does not need the full layer7 capabilties of an {product-title} route.
+====
+
+link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[AWS Load Balancer Controller] is a controller to help manage Elastic Load Balancers for a Kubernetes cluster.
+
+* It satisfies Kubernetes link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress resources] by provisioning link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html[Application Load Balancers (ALB)].
+https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html[Network Load Balancers (NLB)].
+
+Compared with default AWS In Tree Provider, this controller is actively developed with advanced annotations for both link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/ingress/annotations/[ALB] and link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/#lb-type[NLB]. Some advanced usecases are:
+
+* Using native Kubernetes Ingress with ALB
+* Integrate ALB with web application firewall (WAF)
+* Specify NLB source IP ranges
+* Specify NLB internal IP address
+
+link:https://github.com/openshift/aws-load-balancer-operator[AWS Load Balancer Operator] is used to used to install, manage and configure an instance of aws-load-balancer-controller in a OpenShift cluster.
+
+## Prerequisites
+
+* xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[A Multi-AZ ROSA Classic cluster] 
+* BYO VPC cluster
+* AWS CLI
+* OC CLI
+
+## Environment
+
+* Prepare the environment variables:
++
+[source,terminal]
+----
+export AWS_PAGER=""
+export ROSA_CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
+export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed  's|^https://||')
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export SCRATCH="/tmp/${ROSA_CLUSTER_NAME}/alb-operator"
+mkdir -p ${SCRATCH}
+echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+== AWS VPC / Subnets
+
+[NOTE]
+====
+This section only applies to BYO VPC clusters, if you let ROSA create your VPCs you can skip to the following Installation section.
```

Comment:
should probably mention specifically this isn't needed if you follow the linked install doc (rosa-sts-creating-a-cluster-quickly)

Reply:
are we able to collapse this section so it can be opened if needed, but otherwise not shown?  may help with people accidently following the steps.

Reply:
Thank you! I updated the note, and added put that procedure in a collapsible format. Typically I've only see collapsible tables, but this could be an experimental use for that functionality.